pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -101,11 +101,19 @@ async def acker(subscription: str,
                     log.error(
                         'Ack error is unrecoverable, '
                         'one or more messages may be dropped', exc_info=e)
+
+                    async def maybe_ack(ack_id: str) -> None:
```

Comment:
nit: since you've defined this function in a loop, the interpreter will have to spend cycles building and rebuilding this function definition. Always better to pull functions up as much as possible; either to near the top of the `acker` method (eg. outside of the loop), or even a top-level function entirely (ie. and pass in the subscriber_client).

Reply:
I agree with the point, but i'd prefer to keep it here, because logically this is a lambda and it makes most sense when read in this context.